packages/dx-react-chart-material-ui/src/index.js

```diff
@@ -13,4 +13,4 @@ export { ValueAxis } from './plugins/value-axis';
 export { ArgumentAxis } from './plugins/argument-axis';
 
 export { Tooltip } from './plugins/tooltip';
-export { ZoomAndPan } from './plugins/zoom-pan';
+export { ZoomAndPan } from './plugins/zoom-and-pan';
```

Comment:
Should we rename the folder name `zoom-pan` in the `dx-chart-core`?

Reply:
Yes, we should.

Reply:
Renamed.